pyrgg/functions.py

```diff
@@ -232,23 +232,28 @@ def logger(
         print(PYRGG_LOGGER_ERROR_MESSAGE)
 
 
-def time_convert(input_string):
+def time_convert(input_time):
     """
-    Convert input_string from sec to DD,HH,MM,SS format.
+    Convert input_time from sec to DD,HH,MM,SS format.
 
-    :param input_string: input time string in sec
-    :type input_string: str
+    :param input_time: input time in sec
+    :type input_time: float
     :return: converted time as str
     """
-    sec = float(input_string)
-    days, sec = divmod(sec, 24 * 3600)
-    hours, sec = divmod(sec, 3600)
-    minutes, sec = divmod(sec, 60)
+    postfix_dict = {"s": "second", "d": "day", "h": "hour", "m": "minute"}
+    value_dict = {"s": 0, "d": 0, "h": 0, "m": 0}
+    value_dict["s"] = float(input_time)
+    value_dict["d"], value_dict["s"] = divmod(value_dict["s"], 24 * 3600)
+    value_dict["h"], value_dict["s"] = divmod(value_dict["s"], 3600)
+    value_dict["m"], value_dict["s"] = divmod(value_dict["s"], 60)
+    for i in postfix_dict.keys():
+        if value_dict[i] != 1:
```

Comment:
Shouldn't be `if value_dict[i] > 1` so that `00 hours` transforms to `00 hour`?

Reply:
Take a look at : 
1. https://www.minecraftforum.net/forums/off-topic/general-off-topic/405905-how-do-you-say-0-hour-hours
2. https://ell.stackexchange.com/questions/174489/is-it-correct-to-say-0-hours#:~:text=Since%20it's%20not%20being%20said,is%20exactly%20one%20of%20something.